assets/js/usa-search.js

```diff
@@ -1,6 +1,7 @@
 var usasearch_config = { siteHandle:"vets.gov_search" };
 var script = document.createElement("script");
 script.src = "https://search.usa.gov/javascripts/remote.loader.js";
+script.defer = true;
```

Comment:
I think `async` is more often used for this sort of external script, but should be an improvement either way.

Reply:
would you like it as async?  I am happy to :)

Reply:
converted